app/services/cfe_civil/components/irregular_incomes.rb

```diff
@@ -1,15 +1,37 @@
 module CFECivil
   module Components
     class IrregularIncomes < BaseDataBlock
-      delegate :irregular_incomes, to: :legal_aid_application
-
       def call
         {
           irregular_incomes: {
```

Comment:
Have you checked this with a CFE submission?  I'm not sure how it will handle 
```ruby
{
  irregular_incomes: {
    []
  }
}
```
I have seen Civil fail with empty arrays, we may have to not send _anything_ but I'm not sure.  Happy to pair and investigate :+1:

Reply:
Won't this return:
```
{
    irregular_incomes: {
        payments: [],
    }
}
```

That's what the original first spec was returning. I haven't made any changes to that. Either way, pairing to see how it will handle it sounds like a good idea

Reply:
Cool, if you are matching existing tests then it should be fine! :) 
I just remember when I was writing the components that if we sent (I think employment) as 
```ruby
 { employment: [] } 
```
that cfe would error, so we ended up with skipping the whole component :)